README.md

```diff
@@ -21,36 +21,81 @@ Why is it important that we use typing in our database? Simply put, typing allow
+Did we intend age to be represented as a whole-number, a word, or a decimal? If we asked you to add up the ages of all the dogs you could simply convert the 'two' to 2 in your head, but your database can't do that. It doesn't have that ability because the logic involved in converting a word into a number would be dense and inefficient. What about different languages? What about different spellings? Capitalization, typos, or different hyphenation conventions? These are just some reasons this might start to get crazy. In other words, because databases are designed to store large amounts of data, they are very concerned with storing, accessing, and acting upon that data as efficiently, and normally, as possible.
 
 Typing gives us the ability to perform all kinds of operations with predictable results. For instance, the ability to perform Math operations like SUM - i.e. summing integers - doesn't just depend on everything being an integer of some sort but would also expect it. If you tried, for example,  to SUM all of the dogs in the above table, SQLite would actually attempt to convert, or cast, their type to something it can SUM. It would try to convert anything it can to an INTEGER and ignore alpha characters. This can lead to real problems. Without typing, our data might get complicated and messy, and it would be difficult to ask the database questions about large sets of data.
 
 We're going to adhere strictly to only storing data that fits with the datatype we have given to a particular column.
 
-## Data Types
+## Datatypes
 
-Different database systems also have different datatypes available, which are important and useful to know whenever you are dealing with those systems. In SQLite there are five different types we will be dealing with, they are:
+Different database systems also have different datatypes available, which are important and useful to know whenever you are dealing with those systems. In SQLite there are five basic categories of datatypes we will be dealing with, they are:
 
 ```bash
 TEXT
 NUMERIC
 INTEGER
 REAL
-NONE
+BLOB
 ```
 
-For now we'll cover just three types, and later on we'll also learn about a couple other useful types not listed here. Our types:
+These categories each contain several datatypes. Let's explore each category in more detail.
 
 ### TEXT
 
 Any alphanumeric characters which we want to represent as plain text. The body of this paragraph is text. Your name is text. Your email address is a piece of text. Your height, weight, and age, however, are probably not.
 
+Common text datatypes include:
+
+```bash
+	CHARACTER(20)
+	VARCHAR(225)
+	TEXT
```

Comment:
I would have an opinion on which of these is the most useful. In general we just use `TEXT`.